iphone/Maps/UI/PlacePage/Components/PlacePageHeader/PlacePageHeaderViewController.swift

```diff
@@ -50,7 +50,35 @@ extension PlacePageHeaderViewController: PlacePageHeaderViewProtocol {
     }
   }
 
-  func setTitle(_ title: String) {
-    titleLabel?.text = title
+  func setTitle(_ title: String?, secondaryTitle: String?) {
+    guard let title = title,
+          let titleFont = StyleManager.shared.theme?.fonts.medium20 else {
+      titleLabel?.attributedText = nil
+      return
+    }
+
+    let titleAttributes: [NSAttributedString.Key: Any] = [
+      .font: titleFont,
+      .foregroundColor: UIColor.blackPrimaryText()
+    ]
+
+    let attributedText = NSMutableAttributedString(string: title, attributes: titleAttributes)
+
+    guard let secondaryTitle = secondaryTitle,
+          let secondaryTitleFont = StyleManager.shared.theme?.fonts.medium16 else {
```

Comment:
The same here: let's crash instead of hiding a bug if the font is not available.

Reply:
Ok, changed to force-unwrap style.